spec/unit/sources_manager_spec.rb

```diff
@@ -294,9 +295,16 @@ module Pod
 
     describe 'Updating Sources' do
       extend SpecHelper::TemporaryRepos
+      
+      after do
+        WebMock.reset!
+      end
 
       it 'updates source backed by a git repository' do
         set_up_test_repo_for_update
+        WebMock::API.stub_request(:get, "https://api.github.com/repos/cocoapods/specs/commits/master")
```

Comment:
I wasn't sure on this at the time (wasn't sure if I was going to need to 304 response for the test), but will update it now.